main.py

```diff
@@ -10,7 +10,7 @@
 # Qt imports
 from PyQt5 import QtWidgets, QtGui, uic, QtCore
 
-import streamlink
+from streamlink import Streamlink, exceptions as streamlink_exceptions
```

Comment:
Is this really necessary though? Going from `streamlink.exceptions` to `streamlink_exceptions` doesn't feel like an huge improvement to me, but I could be wrong ^^